app/frontend/wallet/shelley/shelley-js-crypto-provider.ts

```diff
@@ -18,14 +18,15 @@ import {
 
 import {isShelleyPath} from './helpers/addresses'
 import CachedDeriveXpubFactory from '../helpers/CachedDeriveXpubFactory'
+import {BIP32Path, CryptoProvider, CryptoProviderFeatures, HexString} from '../../types'
+import NamedError from '../../helpers/NamedError'
 
-type HexString = string & {__typeHexString: any}
-
-const ShelleyJsCryptoProvider = ({
+const ShelleyJsCryptoProvider = async ({
   walletSecretDef: {rootSecret, derivationScheme},
   network,
   config,
-}) => {
+  // eslint-disable-next-line require-await
+}): Promise<CryptoProvider> => {
```

Comment:
Is this actually returning Promise?

Reply:
I changed it to async, so yes it does, but the eslint doesn't like that it doesn't have an await expression and it doesn't have to be async. I changed it because the other crypto providers are async, so afaik, either all crypto providers must async or `ShelleyCryptoProviderFactory` needs to be async.